### PR TITLE
Update PlayBilling version to V6

### PR DESCRIPTION
**Pull request by orbital17**
WIP: API updates for Play Billing V6 migration

### CODE REVIEW

**Review by OlegNitz — APPROVED** [review]

**Review by OlegNitz — APPROVED** [review]

File: playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/provider/PaymentActivity.java [hunk 1]
```diff
@@ -122,13 +122,7 @@ private void onSkusQueried(BillingResult result, List<SkuDetails> skus) {
...
 
     private void launchPriceChangeConfirmationFlow(SkuDetails sku) {
-        mWrapper.launchPriceChangeConfirmationFlow(this, sku, result -> {
-            if (result.getResponseCode() == BillingClient.BillingResponseCode.OK) {
-                setResultAndFinish(PaymentResult.priceChangeSuccess());
-            } else {
-                fail("Price change confirmation flow ended with result: " + result);
-            }
-        });
+        fail("Price change confirmation flow is not supported");
```
Comment (PEConn) [inline playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/provider/PaymentActivity.java:125]: I think we can fail earlier. At the moment, the `PaymentActivity` is launched, it connects to Play Billing (which is async) and queries the SKUs (which is async) then hits this, but we know in `PaymentActivity#onCreate` if the request is for a price change confirmation flow.

I think you could instead add `if (mMethodData.isPricechangeConfirmation) { fail(...) }` as soon as we create the MethodData.

**Review by PEConn — COMMENTED** [review]
Looks good, just one thought.

**Review by PEConn — APPROVED** [review]

**Comment by PEConn** [comment]
To check, you've run the tests locally and they still pass, right?

**Comment by orbital17** [comment]
> To check, you've run the tests locally and they still pass, right?

Yes, tests pass locally

**Pull request merged**